packages/react/src/components/Masthead/Masthead.js

```diff
@@ -138,51 +138,59 @@ const Masthead = ({ navigation, ...mastheadProps }) => {
           <div className={`${prefix}--masthead__l0`}>
             <Header aria-label="IBM" data-autoid={`${stablePrefix}--masthead`}>
               <SkipToContent />
-              <HeaderMenuButton
-                aria-label="Open menu"
-                data-autoid={`${stablePrefix}--masthead__hamburger`}
-                onClick={onClickSideNavExpand}
-                isActive={isSideNavExpanded}
-              />
+
+              {navigation && (
+                <HeaderMenuButton
+                  aria-label="Open menu"
+                  data-autoid={`${stablePrefix}--masthead__hamburger`}
+                  onClick={onClickSideNavExpand}
+                  isActive={isSideNavExpanded}
+                />
+              )}
 
               <IbmLogo />
 
               <div className={`${prefix}--header__search ${hasPlatform}`}>
-                {navigation !== false ? (
+                {navigation && (
                   <MastheadTopNav
                     {...mastheadProps}
                     navigation={mastheadData}
                   />
-                ) : null}
-                <MastheadSearch />
+                )}
+                {mastheadProps.hasSearch && <MastheadSearch />}
```

Comment:
@kennylam I couldn't find defaultProps for these, these should be true by default right?

Reply:
@jeffchew Doh! Yes you are right 👍